AppCenter/AppCenter/Internals/MSAppCenterInternal.h

```diff
@@ -42,6 +42,8 @@ static NSString *const kMSDisableAll = @"All";
 
 @property(nonatomic) NSNumber *requestedMaxStorageSizeInBytes;
 
+@property(nonatomic, getter=isEnabled) BOOL enabled;
```

Comment:
I know other properties are not documented (was a mistake) but please document this one.